src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -115,6 +115,27 @@ protected function redirectToRoute(string $route, array $parameters = [], int $s
         return $this->redirect($this->generateUrl($route, $parameters), $status);
     }
 
+    /**
+     * Returns a RedirectResponse to last accessed url.
+     *
+     * @author Temuri Takalandze (ABGEO) <takalandzet@gmail.com>
+     *
+     * @final
+     */
+    protected function redirectToBack(): RedirectResponse
+    {
+        // Get current request.
+        $request = $this->container
+            ->get('request_stack')
+            ->getCurrentRequest();
+
+        // Get referer URL.
+        $referer = $request->headers->get('referer');
+
+        // Redirect to referer url or /.
```

Comment:
All these comments are totally useless. The code is clear enough to get the same info than what they say. We don't want to maintain such comments on each statement.